demo/index.html

```diff
@@ -57,7 +57,7 @@ <h1>IPA-JFK</h1>
 You can obtain the source code from GitHub:
 https://github.com/b1f6c1c4/IPA-JFK</pre>
 
-    <script src="main.js"></script>
+    <script src="main.js" type="module"></script>
```

Comment:
`demo/main.js` is NOT a JavaScript module. Why include `type="module"` here? :thinking:

Reply:
Was trying to import other libraries and `Eslint` panicked: `Parsing error: 'import' and 'export' may appear only with 'sourceType: module'`

Reply:
Update: migration from deprecated `parcel-bundler` to `parcel` requires `type="module"`. See also [Parcel 2 RC](https://v2.parceljs.org/blog/rc0/).